src/server/pegasus_server_impl.cpp

```diff
@@ -755,11 +784,21 @@ void pegasus_server_impl::on_multi_get(const ::dsn::apps::multi_get_request &req
 
         std::unique_ptr<rocksdb::Iterator> it;
         bool complete = false;
+        uint64_t iteration_time = dsn_now_ns();
```

Comment:
iteration_start_time. 

iteration_time confuses with the time duration rather than the timestamp.

Reply:
iteration_time is duration time, not iteration_start_time, it will be updated when checking time exceed limit.